pages/members/[slug].js

```diff
@@ -122,7 +139,16 @@ const MemberPage = ({ member, loadError }) => {
                   <FontAwesomeIcon icon={faEnvelope}/>
                 </a>
               }
-            </h1>
+            </h3>
+            
+            <div className='mt-4'>
+              {/* <p>This is where the tagline description is placed.</p> */}
```

Comment:
I added the `tagline` field to the user so you can now display it with possibility to edit just like the `user.about` field